Documentation/Index.rst

```diff
@@ -0,0 +1,92 @@
+.. _start:
+
+==================
+Tracking Extension
+==================
+
+A very simple server side tracking extension.
+Initially developed for demonstration purposes.
+Soon requested by customers in order to have privacy focused minimal tracking inside of TYPO3.
+
+The extension provides built in tracking.
+All requests are tracked right within TYPO3 as custom records.
+
+The extension also delivers widgets for EXT:dashboard to visualize tracked information.
+
+.. _goal:
+
+Goal
+----
+
+This extension only provides very basic features and is not intended to resolve complex solutions like Google Analytics.
+Yet it should provide the very minimum requirements to remove the need for such complex solution on small websites.
+
+.. figure:: /Images/Widgets.png
+    :align: center
+
+    Figure 1-1: Screenshot of how widgets might look, representing tracked information.
+
+.. figure:: /Images/ListView.png
+    :align: center
+
+    Figure 1-2: Screenshot of TYPO3 list view, showing records of tracked data.
+
+Integrators should be able to configure more or less everything.
+From collection to displaying via widgets.
+
+Features
+--------
+
+The extension allows to track :ref:`pageviews`,
```

Comment:
The reference `pageviews` does not exists and nothing is linked here.

You can see it in your preview of the documentation.